src/index.ts

```diff
@@ -24,6 +24,27 @@ export abstract class AlpineComponent {
     [key: string]: any;
 }
 
+declare type Alpine = {
+	version: string;
+	pauseMutationObserver: boolean;
+	magicProperties: { [name: string]: CallableFunction };
+	ignoreFocusedForValueBinding: boolean;
+	pauseMutationObserver: boolean;
+	onComponentInitializeds: Array<any>;
+	onBeforeComponentInitializeds: Array<any>;
+	onComponentInitializeds: (callback: CallableFunction) => void;
```

Comment:
I think the function shouldn't be plural, i.e. `onComponentInitialized`

Reply:
yes right sorry!